skype/skype-ps/skype/Grant-CsTeamsIPPhonePolicy.md

```diff
@@ -0,0 +1,190 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+Module Name: Skype for Business Online
+online version: https://docs.microsoft.com/powershell/module/skype/grant-csteamsipphonepolicy
+applicable: Skype for Business Online
+title: Grant-CSTeamsIPPhonePolicy
+author: tonywoodruff
+ms.author: anwoodru
+ms.reviewer: kponnus
+manager: sandrao
+schema: 2.0.0
+---
+
+# Grant-CsTeamsIPPhonePolicy
+
+## SYNOPSIS
+
+Use the Grant-CSTeamsIPPhonePolicy cmdlet to assign a set of Teams phone policies to a user account or a group of users. Teams phone policies determine the features that are available to users of Teams phones. For example, you might enable the hot desking feature for some users while disabling it for others.

```

Comment:
```suggestion
Use the Grant-CSTeamsIPPhonePolicy cmdlet to assign a set of Teams phone policies to a user account. Teams phone policies determine the features that are available to users of Teams phones. For example, you might enable the hot desking feature for some users while disabling it for others.

```

Could you assign a policy to a group of users using only Grant-CSTeamsIPPhonePolicy?

Reply:
We could through a CSV piping the Grant-CSTeamsIPPhonePolicy cmdlet.